src/Akka.Hosting/AkkaHostingExtensions.cs

```diff
@@ -63,8 +64,23 @@ public static IServiceCollection AddAkka(this IServiceCollection services, strin
             // registers the hosted services and begins execution
             b.Bind();
             
-            // start the IHostedService which will run Akka.NET
-            services.AddHostedService<AkkaHostedService>();
+            if (Util.IsRunningInMaui)
+            {
+                services.AddSingleton<AkkaHostedService>(provider =>
```

Comment:
MAUI does not support hosting, hosted services had to be registered as a singleton service.